### PR TITLE
EWL-4423 SG2 | Create tag atom and tags molecule

### PR DESCRIPTION
**Pull request by chelsiejohnston**
## Ticket(s)

**Jira Ticket**

- [EWL-4423:  | Create tag atom and tags molecule](https://issues.ama-assn.org/browse/EWL-4423)

## Description
Adds a Tag atom and a Tags molecule.

## To Test

- [ ] Atoms > Tag: make sure it looks okay and meets our SG2 standards
- [ ] Molecules > Tags: make sure it looks okay and meets our SG2 standards

## Visual Regressions

no regression testing in SG2 yet

## Relevant Screenshots/GIFs
screen shot 2018-01-17 at 11 48 16 am

## Remaining Tasks

 - [x] review with UX

## Additional Notes
N/A

### CODE REVIEW

File: styleguide/source/assets/scss/02-molecules/_tags.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    display: inline-block;
+  }
+}
+
+.ama__tag {
+  display: inline-block;
+  padding: ($gutter / 4) ($gutter / 2);
+  background: $grey-7;
+  color: $purple;
+}
```
Comment (designerneil) [inline styleguide/source/assets/scss/02-molecules/_tags.scss]: need a return here so github is happy

File: styleguide/source/_patterns/02-molecules/tags.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    {
+      "text": "Lorem ipsum",
+      "url": "http://www.google.com"
+    },
+    {
+      "text": "Dolor sit",
+      "url": "http://www.yahoo.com"
+    }
+  ]
+}
```
Comment (designerneil) [inline styleguide/source/_patterns/02-molecules/tags.json]: empty return here for github

File: styleguide/source/_patterns/01-atoms/tag.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "text": "Example text",
+  "url": "http://www.example.com"
+}
```
Comment (designerneil) [inline styleguide/source/_patterns/01-atoms/tag.json]: blank line needed

File: styleguide/source/assets/scss/02-molecules/_tags.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  li {
+    display: inline-block;
+  }
+}
+
+.ama__tag {
+  display: inline-block;
+  padding: ($gutter / 4) ($gutter / 2);
+  background: $grey-7;
```
Comment (designerneil) [inline styleguide/source/assets/scss/02-molecules/_tags.scss]: i think this variable was renamed to $gray-7

**Comment by labbydev** [comment]
@chelsiejohnston this looks good just a couple quick line ending updates

File: styleguide/source/assets/scss/00-base/_colors.scss [hunk 5]
```diff
@@ -33,5 +33,5 @@ $hoverBlack: #333333;
 $black: #000000;
 $grey-64: #5C5C5C;
 $grey-50: #7F7F7F;
-$grey-7: #EDEDED;
+$gray-7: #EDEDED;
 $white: #ffffff;
```
Comment (labbydev) [inline styleguide/source/assets/scss/00-base/_colors.scss]: Line ending

**Review by labbydev — APPROVED** [review]
This looks good, just fix the last line in `_colors.scss` and resolve the merge conflicts and this is good to go.

**Review by labbydev — APPROVED** [review]

**Pull request merged**